dev/com.ibm.ws.opentracing/src/com/ibm/ws/opentracing/OpentracingUserFeatureAccessService.java

```diff
@@ -38,9 +38,22 @@
 
     private static OpentracingTracerFactory opentracingTracerFactory;
 
+    private static boolean factoryFirstUse = true;
+
     @Reference
     public void setOpentracingTracerFactory(OpentracingTracerFactory opentracingTracerFactory) {
         OpentracingUserFeatureAccessService.opentracingTracerFactory = opentracingTracerFactory;
+        factoryFirstUse = false;
+
+        if (opentracingTracerFactory != null) {
+            if (TraceComponent.isAnyTracingEnabled() && tc.isInfoEnabled()) {
+                Tr.info(tc, "OPENTRACING_NEW_TRACERFACTORY", opentracingTracerFactory.getClass().getName());
+            }
+        } else {
+            if (TraceComponent.isAnyTracingEnabled() && tc.isErrorEnabled()) {
```

Comment:
Remove trace guards for Tr.error.  It isn't correct to require isAnyTracingEnabled in order to log errors